README.md

```diff
@@ -59,6 +59,36 @@ Code in **Play controllers** should be limited to brokering interaction between
 
 Code in **ebean models** should be limited to brokering interaction between the server's business logic and the database. Code in models should never directly implement business logic concerns.
 
+### Routing and controller methods
+
+APIs should follow [REST](https://en.wikipedia.org/wiki/Representational_state_transfer) when possible with the appropriate HTTP verb. Routing should be [resource-oriented](https://www.oreilly.com/library/view/restful-web-services/9780596529260/ch04.html) ([relevant AIP](https://google.aip.dev/121)).
+
+#### HTML routing convention
+
+For a resource called "programs" that implements the standard actions via HTML requests the routes would be:
```

Comment:
This is small, but we might want to add style guidance for multi-word URL segments, e.g., use hyphens to separate words in the same segment.

Reply:
Thanks! Added a sentence above: `Path names should be [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Special_case_styles).`